lib/chef/resource/chef_client_config.rb

```diff
@@ -208,6 +208,11 @@ def string_to_symbol(prop_val)
         description: %q(An array of hashes that contain a report handler class and the arguments to pass to that class on initialization. The hash should include `class` and `argument` keys where `class` is a String and `argument` is an array of quoted String values. For example: `[{'class' => 'MyHandler', %w('"argument1"', '"argument2"')}]`),
         default: []
 
+      property :rubygems_url, [String, Array],
+        description: "The location to source rubygems. It can be set to a string or array of strings for URIs to set as rubygems sources. This allows individuals to setup an internal mirror of rubygems for “airgapped” environments.",
+        default: "",
```

Comment:
this probably isn't necessary.  the default should probably be nil, either way it gets handled correctly by the code in the template, but nil is a better "unset" value.

Reply:
@lamont-granquist Is your recommendation to set `default: nil` or to remove the `default:` entirely?

Reply:
Set it to nil please

Reply:
You can omit `default` and it'll gain a default of nil.  That'd be preferable.

Reply:
@lamont-granquist Done.